### DISCUSSION TITLE
vLLM for SBERT

### DISCUSSION THREAD

**Discussion by david-koleckar** [vllm-project/vllm#1575] [Q&A]
Hi there,   
would it be possible to add sbert model (eg. [paraphrase-multilingual-mpnet-base-v2](https://huggingface.co/sentence-transformers/paraphrase-multilingual-mpnet-base-v2)) to vLLM as described in [adding a new model](https://vllm.readthedocs.io/en/latest/models/adding_model.html)?  
Thank you, David

**Comment by WoosukKwon** [comment]
Hi @david-koleckar, thanks for the proposal. For now, we are not planning to support SBERT model or other embedding/encoder models. vLLM is optimized for generative (auto-regressive) Transformers. Its core technologies like PagedAttention cannot be used for embedding/encoder models.
